### PR TITLE
Task DVD-185

### PR DESCRIPTION
**Pull request by TheoWolf**

### CODE REVIEW

**Comment by Xubes** [comment]
Working as intended. Checked both JavaFX and FFMPEG on Windows.

File: src/main/java/org/datavyu/util/ClockTimer.java [hunk 1]
```diff
@@ -142,6 +142,15 @@ public synchronized double getStreamTime() {
         return (long) clockTime + minTime;
     }
 
+    /**
+     * Get the current Clock time
```
Comment (fraudies) [inline src/main/java/org/datavyu/util/ClockTimer.java]: Please add a note that this is the current time within the tolerance of the periodic sync

File: src/main/java/org/datavyu/views/VideoController.java [hunk 2]
```diff
@@ -1522,9 +1522,10 @@ public void setRegionOfInterestAction() {
     @SuppressWarnings("unused")  // Called through actionMap
     public void goBackAction() {
         try {
-            long oldTime = -CLOCK_FORMAT.parse(goBackTextField.getText()).getTime();
-            logger.info("Jump back by " + oldTime);
-            clockTimer.setForceTime(oldTime);
+            long backTime = -CLOCK_FORMAT.parse(goBackTextField.getText()).getTime();
+            long newTime = (long) clockTimer.getClockTime() + backTime;
```
Comment (fraudies) [inline src/main/java/org/datavyu/views/VideoController.java]: Please check if we need to guard against jumping beyond the min/max here?

Comment (TheoWolf) [reply src/main/java/org/datavyu/views/VideoController.java]: I did check, setForceTime check if the new time is in between the min and max before seeking.

File: src/main/java/org/datavyu/util/ClockTimer.java (hunk 1 above)
Comment (TheoWolf) [reply src/main/java/org/datavyu/util/ClockTimer.java]: OK.

**Pull request merged**